qa/TL1_nodeps_build/test.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash -e
+
+pip_packages=""
+
+do_once() {
+  apt-get update
```

Comment:
I am not very happy that we have to build CMake every time this test runs. However it's only L1

Reply:
Maybe we can switch to prebuilds https://cmake.org/download/ ?

Reply:
done